examples/acquiring_data/querying_the_GOESeventlist.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
```

Comment:
What about the filename `querying_the_GOES_event_list.py `?

Reply:
yep - done!